backend/src/main/java/com/capstone/moneytree/controller/MoneyTreeController.java

```diff
@@ -13,6 +13,6 @@
 @Retention(RetentionPolicy.RUNTIME)
 @RestController
 @RequestMapping("/api/v1")
-@CrossOrigin(value = "http://localhost")
+@CrossOrigin(origins = "http://127.0.0.1")
```

Comment:
We also need to be able to test with CORS locally. Add this other origins too like follows:
@CrossOrigin(origins = {"http://localhost:4200", "http://127.0.0.1"})

Reply:
nice catch